Ran `sudo make demo_content` in this branch, errored out as follows:

```
sed -i 's/^nopassword.*/password\:  /g' islandora_workbench/islandora_workbench_demo_content/example_content.yml
cd islandora_workbench && docker build -t workbench-docker .
[+] Building 134.9s (12/12) FINISHED                                                                                                                          
 => [internal] load build definition from Dockerfile                                                                                                     0.0s
 => => transferring dockerfile: 802B                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/python:3.10.6                                                                                         0.8s
 => [internal] load build context                                                                                                                        3.4s
 => => transferring context: 232.55MB                                                                                                                    3.4s
 => CACHED [1/7] FROM docker.io/library/python:3.10.6@sha256:745efdfb7e4aac9a8422bd8c62d8bc35a693e8979a240d29677cb03e6aa91052                            0.0s
 => [2/7] ADD . /workbench/                                                                                                                              0.7s
 => [3/7] WORKDIR /workbench                                                                                                                             0.0s
 => [4/7] RUN python -m pip install setuptools                                                                                                           6.5s
 => [5/7] RUN pip install urllib3>=1.21.1                                                                                                                2.9s
 => [6/7] RUN pip install libmagic                                                                                                                       5.2s 
 => [7/7] RUN python setup.py install                                                                                                                  111.8s 
 => exporting to image                                                                                                                                   3.4s 
 => => exporting layers                                                                                                                                  3.4s 
 => => writing image sha256:8fe240d8a4b79adb901a66a1f02ff60fd9d314687dafa04c8b4c334c1cd6a4d5                                                             0.0s 
 => => naming to docker.io/library/workbench-docker                                                                                                      0.0s 
cd islandora_workbench && docker run -it --rm --network="host" -v /home/mark/hacking/docker/isle-dc:/workbench --name my-running-workbench workbench-docker bash -lc "./workbench --config /workbench/islandora_workbench_demo_content/example_content.yml"                                                                 
bash: line 1: ./workbench: No such file or directory
make: *** [Makefile:382: demo_content] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -372,6 +372,7 @@ local: generate-secrets
 .PHONY: demo_content
 #.SILENT: demo_content
 ## Helper function for demo sites: do a workbench import of sample objects
+demo_content: QUOTED_CURDIR = "$(CURDIR)"
 demo_content:
 	# fetch repo that has csv and binaries to data/samples
 	# if prod do this by default
@@ -379,7 +380,7 @@ demo_content:
 	cd islandora_workbench ; cd islandora_workbench_demo_content || git clone https://github.com/DonRichards/islandora_workbench_demo_content
 	$(SED_DASH_I) 's/^nopassword.*/password\: $(shell cat secrets/live/DRUPAL_DEFAULT_ACCOUNT_PASSWORD) /g' islandora_workbench/islandora_workbench_demo_content/example_content.yml
 	cd islandora_workbench && docker build -t workbench-docker .
-	cd islandora_workbench && docker run -it --rm --network="host" -v $(shell pwd):/workbench --name my-running-workbench workbench-docker bash -lc "./workbench --config /workbench/islandora_workbench_demo_content/example_content.yml"
+	cd islandora_workbench && docker run -it --rm --network="host" -v $(QUOTED_CURDIR)/islandora_workbench:/workbench --name my-running-workbench workbench-docker bash -lc "./workbench --config /workbench/islandora_workbench_demo_content/example_content.yml"
 	$(MAKE) reindex-solr
 
 .PHONY: clean

